src/amber/server/settings.cr

```diff
@@ -12,7 +12,7 @@ module Amber
     class_property port_reuse : Bool
     class_property log : ::Logger
     class_property color : Bool = true
-    class_property env = ENV["AMBER_ENV"]? || "development"
+    class_property env = ENV["AMBER_ENV"]? || "development" #This doesn't need to be here.
```

Comment:
Can we remove the inline comment and put it above the code. Also what exactly does not need to be there?

Reply:
In fact we can get rid of this line completely